algorithms/glued_trees/glued_trees.ipynb

```diff
@@ -0,0 +1,438 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.        write_qmod(model, 'circuits/'+filename)
</pre><p>There is not need to save the qmod to a file, as there are many qmods</p><p>I suggest instead create a single representative qmod, and save it under <code>glued_trees.qmod</code> in the main directory, so it can be viewed from the IDE model page </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/151/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>